iz_helpers/static_variables.py

```diff
@@ -5,9 +5,9 @@
 default_prompt = """
 {
     "prompts":{
-        "headers":["outpaint steps","prompt"],
+        "headers":["outpaint steps","prompt","image location"],
         "data":[
-            [0,"Huge spectacular Waterfall in a dense tropical forest,epic perspective,(vegetation overgrowth:1.3)(intricate, ornamentation:1.1),(baroque:1.1), fantasy, (realistic:1) digital painting , (magical,mystical:1.2) , (wide angle shot:1.4), (landscape composed:1.2)(medieval:1.1), divine,cinematic,(tropical forest:1.4),(river:1.3)mythology,india, volumetric lighting, Hindu ,epic,  Alex Horley Wenjun Lin greg rutkowski Ruan Jia (Wayne Barlowe:1.2) <lora:epiNoiseoffset_v2:0.6> "]
+            [0,"Huge spectacular Waterfall in a dense tropical forest,epic perspective,(vegetation overgrowth:1.3)(intricate, ornamentation:1.1),(baroque:1.1), fantasy, (realistic:1) digital painting , (magical,mystical:1.2) , (wide angle shot:1.4), (landscape composed:1.2)(medieval:1.1), divine,cinematic,(tropical forest:1.4),(river:1.3)mythology,india, volumetric lighting, Hindu ,epic,  Alex Horley Wenjun Lin greg rutkowski Ruan Jia (Wayne Barlowe:1.2) <lora:epiNoiseoffset_v2:0.6> ","C:\\path\\to\\image.png"]
```

Comment:
Instead of a filesystem path, lets consider a DATAURL. Browser and fileystem are not good friends.
and, when sharing promps none has the images being referenced by this path.
the dataurl is base64 encoded and keeps the image embedded. sure, the json grows. but thats ok

Reply:
That would be a huge JSON file. Maybe we could make it compatible with both as it is a string. We can be confident that we are in a local installed environment, so the filesystem path works in my projects.

I am sure we can make or find a function that reads, filesystem, URL and DataURL.

Reply:
Done

Reply:
[infinite-zoom-prompts-12-test-with-images.json.txt](https://github.com/v8hid/infinite-zoom-automatic1111-webui/files/11307527/infinite-zoom-prompts-12-test-with-images.json.txt)
Successfully Tested this with :
 a file path like "F:\\sd\\stable-diffusion-webui\\outputs\\txt2img-images\\myimage.png"
-  a URL like "http:www.openai.com/images/myimage.png" 
- a DataURL like "data:png;base64: codebla.."

Reply:
> We can be confident that we are in a local installed environment

as the maintainer of the repo I'm using a server because I don't have a local GPU like so many other people, so we should not consider everyone on local env

Reply:
Could you use a 4090 or do you prefer to work remotely?

I am unsure how you accomplished being on a server at a reasonable price.